rwengine/src/data/GameTexts.hpp

```diff
@@ -28,6 +28,19 @@ namespace GameStringUtil
 GameString fromString(const std::string& str);
 }
 
+/**
+ * Since the encoding of symbols is arbitrary, these constants should be used in
+ * hard-coded strings containing symbols outside of the ASCII-subset supported by
+ * all fonts
+ */
+namespace GameSymbols
+{
+	static constexpr GameStringChar Money  = '[';
+	static constexpr GameStringChar Heart  = '{';
+	static constexpr GameStringChar Armour = '{';
+	static constexpr GameStringChar Star   = ']';
+}
```

Comment:
No. I'm not sure how that could be achieved without inheriting from basic_string, which is bad, or proxying calls.